e2e-tests/functions

```diff
@@ -466,3 +469,98 @@ check_passwords_leak() {
 		collect_logs $OPERATOR_NS
 	fi
 }
+
+calculate_backup_destination() {
```

Comment:
I would name like `get_backup_destination` since we don't calculate anything here.

Reply:
https://github.com/percona/percona-postgresql-operator/pull/524/commits/8f575870c5e566a65e8008634c4f898287498369